yumi:
  hiqp_joint_velocity_controller:
    #####      JOINT LIMITATIONS    #####
     hiqp_preload_jnt_limits:
      - link_frame: yumi_link_1_r
        limitations: [15.0, 1.4835, -2.8, 2.8, 20.0, 10.0] #ddq_max, dq_max, lower joint limit, upper joint limit, P gain, D gain
      - link_frame: yumi_link_2_r
        limitations: [15.0, 1.4835, -2.4, 0.6, 20.0, 10.0] 
      - link_frame: yumi_link_3_r
        limitations: [15.0, 1.7453, -2.8, 2.8, 20.0, 10.0]
      - link_frame: yumi_link_4_r
        limitations: [15.0, 1.3090, -2.0, 1.3, 20.0, 10.0]
      - link_frame: yumi_link_5_r
        limitations: [15.0, 2.2689, -4.9, 4.9, 20.0, 10.0] 
      - link_frame: yumi_link_6_r
        limitations: [15.0, 2.2689, -1.4, 2.3, 20.0, 10.0]
      - link_frame: yumi_link_7_r
        limitations: [15.0, 2.2689, -3.9, 3.9, 20.0, 10.0]
      - link_frame: yumi_link_1_l
        limitations: [15.0, 1.4835, -2.8, 2.8, 20.0, 10.0] #ddq_max, dq_max, lower joint limit, upper joint limit, P gain, D gain
      - link_frame: yumi_link_2_l
        limitations: [15.0, 1.4835, -2.4, 0.6, 20.0, 10.0] 
      - link_frame: yumi_link_3_l
        limitations: [15.0, 1.7453, -2.8, 2.8, 20.0, 10.0]
      - link_frame: yumi_link_4_l
        limitations: [15.0, 1.3090, -2.0, 1.3, 20.0, 10.0]
      - link_frame: yumi_link_5_l
        limitations: [15.0, 2.2689, -4.9, 4.9, 20.0, 10.0] 
      - link_frame: yumi_link_6_l
        limitations: [15.0, 2.2689, -1.4, 2.3, 20.0, 10.0]
      - link_frame: yumi_link_7_l
        limitations: [15.0, 2.2689, -3.9, 3.9, 20.0, 10.0] 
